src/saler/enums/enums.ts

```diff
@@ -0,0 +1,6 @@
+enum TypeSaler {
+  PrivateTutor = 'privateTutor',
+  LegalEntity = 'legalEntity',
```

Comment:
The names for the types of seller, especially "private tutor", seems to be not very appropriate. Better to use more clear,  not confusing names